source/usage-examples/count.txt

```diff
@@ -3,3 +3,47 @@ Count Documents
 ===============
 
 .. default-domain:: mongodb
+
+You can count the number of documents in a collection using the
+``EstimatedDocumentCount()`` and ``CountDocuments()`` methods. 
+
+The following example uses the ``EstimatedDocumentCount()`` method to
+count the number of documents in the ``movies`` collection. Then, the
+example used the ``CountDocuments()`` method to count the number of
+documents in the ``movies`` collection where the ``countries`` field is
+"China".
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/count.go
+   :start-after: begin countDocuments
+   :end-before: end countDocuments
+   :emphasize-lines: 4-5
+   :language: go
+   :dedent:
+
+Click here <TODO> to see a fully runnable example.
+
+Expected Result
+---------------
+
+After running the preceding code snippet, you should see the following:
+
+- There are about ``23541`` documents in the ``movies`` collection  
+- There are ``755`` documents in the ``movies`` collection from "China"
+
+.. note::
+
+    The exact number of documents may vary depending on your data set.
+
+Additional Information
+----------------------
+
+For more information on counting documents, see our guide on
+<TODO: Counting Documents>. 
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+- `CountDocuments() <https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.7.1/mongo#Collection.CountDocuments>`__
+- `EstimatedDocumentCount() <https://pkg.go.dev/go.mongodb.org/mongo-driver@v1.7.1/mongo#Collection.EstimatedDocumentCount>`__
```

Comment:
See [here](https://github.com/mongodb/docs-golang/blob/master/source/usage-examples/insertOne.txt#L54) for how we're doing API links now.